### ISSUE TITLE
"outputs by sector" query collapses the output column

### ISSUE THREAD

**Issue by pkyle** [JGCRI/modelinterface#8]
The "outputs by sector" query is currently not writing out the "output" column, instead adding up all outputs from a sector and reporting them together. Because sectors with multiple outputs typically have outputs with different units (e.g., the Corn sector produces Mt of corn and EJ of residue biomass), we don't generally want to add them up. The "outputs by subsector" and "outputs by technology" queries both work fine.

**Comment by DLoughlin** [comment]
I was able to replicate the problem. I'm not entirely sure why this fix works, but when I modify the XPath to eliminate outputs with the name "N/A", it appears to fix the problem. As a little more background, when running the "outputs by tech" query, there were some odd responses for energy service demands with physical units. For example, the commercial lighting results included:
Sector: comm lighting; Subsector: electricity; Output: comm lighting (in peta-lumens)
Sector: comm lighting; Subsector: electricity; Output: N/A (in EJ)

`<xPath buildList="true" dataName="output" group="false" sumAll="false">*[@type='sector']//*[@type='output' and @name!='N/A']/physical-output/node()</xPath>`

**Comment by pkyle** [comment]
Yeah I had noticed a similar thing, that when a technology produces multiple outputs, the "outputs by sector" query no longer adds them up, but just reports them in different rows, though they aren't labeled so it still isn't ideal. Filtering the output to remove "N/A" probably just moves the internal-gains outputs of technologies in the buildings sector (this is the heat released by building interior lighting), but it won't help to clarify or modify the outputs of e.g. CHP systems where the technologies produce multiple outputs.
